pyproject.toml

```diff
@@ -1,3 +1,17 @@
 [build-system]
-requires = ["jupyter_packaging~=0.4.0", "jupyterlab~=2.0", "setuptools>=40.8.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["jupyter_packaging~=0.10,<2", "jupyterlab~=3.1"]
+build-backend = "jupyter_packaging.build_api"
+
+[tool.jupyter-packaging.options]
+skip-if-exists = ["jupyterlab_h5web/labextension/static/style.js"]
+ensured-targets = ["jupyterlab_h5web/labextension/static/style.js", "jupyterlab_h5web/labextension/package.json"]
+
+[tool.jupyter-packaging.builder]
+factory = "jupyter_packaging.npm_builder"
+
+[tool.jupyter-packaging.build-args]
+build_cmd = "build:prod"
+npm = ["jlpm"]
+
+[tool.check-manifest]
+ignore = ["jupyterlab_h5web/labextension/**", "yarn.lock", ".*", "package-lock.json", "CONTRIBUTING.md", "requirements_release.txt", "example.ipynb"]
```

Comment:
`check-manifest` bothered me saying that `"CONTRIBUTING.md", "requirements_release.txt", "example.ipynb"` were not included in the `sdist` bundle.

@t20100 Do you think these files should be included ?

Reply:
can't you ignore them in the `MANIFEST.in`?